### PR TITLE
metrics: Add latency network metrics script

### LINKED ISSUE #4987: metrics: Add network latency metrics

### PR DESCRIPTION
**Pull request by GabyCT**
This PR adds a latency network metrics script so we in the future
enable this in our daily metrics CI.

Fixes #4987

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

**Review by dborquez — APPROVED** [review]
lgtm, thanks @GabyCT

File: metrics/network/latency_kubernetes/latency-network.sh [hunk 1]
```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -x
```
Comment (chavafg) [inline metrics/network/latency_kubernetes/latency-network.sh:7]: we could use `[ -n "$DEBUG" ] && set -x` as other scripts do

**Review by chavafg — APPROVED** [review]
just a small suggestion, but LGTM

File: metrics/network/latency_kubernetes/latency-network.sh (hunk 1 above)
Comment (GabyCT) [reply metrics/network/latency_kubernetes/latency-network.sh:7]: @chavafg yes I can do the change later for that as I will remove the set -x when I enable the CI limits for this measurement but first I want it like this so I can see the general output in the baremetal instance

**Pull request merged**